crates/net/discv5/src/lib.rs

```diff
@@ -729,16 +729,12 @@ mod test {
         node_1.with_discv5(|discv5| discv5.send_ping(node_2_enr.clone())).await.unwrap();
 
         // verify node_1:discv5 is connected to node_2:discv5 and vv
-        let event_2_v5 = stream_2.recv().await.unwrap();
         let event_1_v5 = stream_1.recv().await.unwrap();
+
         assert!(matches!(
             event_1_v5,
             discv5::Event::SessionEstablished(node, socket) if node == node_2_enr && socket == node_2_enr.udp4_socket().unwrap().into()
         ));
-        assert!(matches!(
-            event_2_v5,
-            discv5::Event::SessionEstablished(node, socket) if node == node_1_enr && socket == node_1_enr.udp4_socket().unwrap().into()
-        ));
```

Comment:
this is an internal discv5 event change so this established event is no longer emitted